script/character_scraper.rb

```diff
@@ -34,7 +34,7 @@
   end
 
   unless CharactersGallery.where(gallery_id: gallery.id, character_id: character.id).exists?
-    CharactersGallery.create(character_id: character.id, gallery_id: gallery.id)
+    CharactersGallery.create!(character_id: character.id, gallery_id: gallery.id)
   end
```

Comment:
Think this thing can actually condense into:
```ruby
CharactersGallery.find_or_create_by!(character_id: character.id, gallery_id: gallery.id)
```

(But it's very old code, so it doesn't much matter.)